lib/BankAccount.php

```diff
@@ -80,17 +83,25 @@ public static function update($_id, $_params = null, $_options = null)
         throw new Error\InvalidRequest($msg, null);
     }
 
-   /**
-     * @param array|null $params
+    /**
+     * @param array|null        $params
      * @param array|string|null $options
      *
      * @return BankAccount The verified bank account.
+     * @throws Error\Api
+     * @throws Error\ApiConnection
+     * @throws Error\Authentication
+     * @throws Error\Card
+     * @throws Error\Idempotency
+     * @throws Error\InvalidRequest
+     * @throws Error\Permission
+     * @throws Error\RateLimit
      */
     public function verify($params = null, $options = null)
     {
-        $url = $this->instanceUrl() . '/verify';
-        list($response, $opts) = $this->_request('post', $url, $params, $options);
-        $this->refreshFrom($response, $opts);
-        return $this;
+         $url = $this->instanceUrl() . '/verify';
+         list($response, $opts) = $this->_request('post', $url, $params, $options);
+         $this->refreshFrom($response, $opts);
+         return $this;
```

Comment:
I think the indentation here was correct before. Can you dedent this block by one space?